sdk/servicebus/service-bus/rollup.test.config.js

```diff
@@ -0,0 +1,73 @@
+import cjs from "@rollup/plugin-commonjs";
+import inject from "@rollup/plugin-inject";
+import json from "@rollup/plugin-json";
+import multiEntry from "@rollup/plugin-multi-entry";
+import nodeResolve from "@rollup/plugin-node-resolve";
+import replace from "@rollup/plugin-replace";
+import shim from "rollup-plugin-shim";
+import sourcemaps from "rollup-plugin-sourcemaps";
+import { makeConfig, makeBrowserTestConfig } from "@azure/dev-tool/shared-config/rollup";
+
+const inputs = makeConfig(require("./package.json"));
+
+if (!process.env.ONLY_NODE) {
+  // service-bus has many dependencies that we do not
+  // want to bring into the shared rollup config, so
+  // replace the original test config with a patched one
+  inputs[1] = makeBrowserTestConfigPatch();
+}
+
+function makeBrowserTestConfigPatch() {
+  const config = { ...makeBrowserTestConfig(require("./package.json")) };
+  config.plugins = [
+    multiEntry({ exports: false }),
+    sourcemaps(),
+    replace(
+      // ms-rest-js is externalized so users must include it prior to using this bundle.
```

Comment:
does this ms-rest-js comment still apply? Probably should remove it.

Reply:
Doesn't apply, thank you! I got rid of it